aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -77,6 +77,35 @@ module aptos_std::simple_map {
         vector::push_back(&mut map.data, Element { key, value });
     }
 
+    public fun to_vec<Key: store, Value: store>(
```

Comment:
Can you add documentations for all these new public functions?

Reply:
removed them they were relics, I removed destroy from this PR